cli/src/cmd/forge/script/broadcast.rs

```diff
@@ -225,13 +232,29 @@ impl ScriptArgs {
         libraries: Libraries,
         decoder: &mut CallTraceDecoder,
         mut script_config: ScriptConfig,
-        mut verify: VerifyBundle,
+        verify: VerifyBundle,
```

Comment:
Tried to make it clearer. Lmk if after this recente changes, you'd really separate it further... I feel like it's quite contained now. 

Eventually (when library linking is done), the goal is that the single deployment conditionals mostly disappear, since single deployment is a multi-deployment of one...